ccmlib/scylla_node.py

```diff
@@ -1438,11 +1438,13 @@ def do_invoke(sstables):
 
     def dump_sstables(self,
                       keyspace: str,
-                      column_family: str) -> List[Dict[str, Any]]:
+                      column_family: str,
+                      datafile: Optional[str] = None) -> List[Dict[str, Any]]:
```

Comment:
Lets make this a list instead, it is more forward compatible, and you already use a list internally anyway.

Reply:
sure. i don't see the needs so far. but yeah, it's more future-proof.